app/sell/index.js

```diff
@@ -1,14 +1,138 @@
+/* eslint-disable no-undef */
 import React from 'react'
-import { Text } from 'react-native-paper'
+import {StyleSheet, View, Text, Dimensions, ScrollView } from 'react-native'
+import { Link } from 'expo-router'
 
 import Page from '@components/Page'
 
+const { width: screenWidth } = Dimensions.get('window');
+
 const Sell = () => {
+    const options = [
+        {
+            label: 'Chick-fil-A',
+            time: '2024-09-14T19:30:00'
+        },
+        {
+            label: 'Panda',
+            time: '2024-09-14T19:30:00'
+        },
+        {
+            label: 'Dunkin',
+            time: '2024-09-14T19:35:00'
+        },
+        {
+            label: "Canes",
+            time: '2024-09-14T19:40:00'
+        },
+        {
+            label: "Pres-Deli",
+            time: '2024-09-14T19:40:00'
+        },
+        {
+            label: "Julias",
+            time: '2024-09-14T20:00:00'
+        },
+        {
+            label: "Wendy's",
+            time: '2024-09-14T20:00:00'
+        },
+    ]
+
+    const formatTimeWithIntl = (timeString) => {
+        const date = new Date(timeString);  // Convert the ISO time string to a Date object
+    
+        // Use Intl.DateTimeFormat with options for time only (hours, minutes, AM/PM)
+        const formatter = new Intl.DateTimeFormat('en-US', {
+            hour: 'numeric',
+            minute: 'numeric',
+            hour12: true,  // This ensures the time is in 12-hour format with AM/PM
+        });
```

Comment:
The semicolons and trailing commas here are indicative that your Prettier extension is not working properly. We use Prettier to guarantee a consistent code formatting style. Ideally, it should (1) automatically reformat your file whenever you save and (2) automatically reformat all files when you commit. Clearly, something isn't quite working. See if you can get it set up and then reformat this code.

First, make sure you've actually installed the required extensions in the ReadMe. Then, you may have to set Prettier as your default formatter in your VSCode settings and then restart VSCode. If that doesn't work, you can check the "output" tab on VSCode to see what the error is.

Reply:
fixed i think